pkg/cmd/root/root.go

```diff
@@ -28,6 +56,22 @@ func NewRootCmd(cfg *config.Config) *cobra.Command {
 		},
 	}
 
+	rootCmd.PersistentFlags().StringVarP(
+		&cfg.LogLevel,
```

Comment:
This config was meant to represent all the persisted options. I'm not sure if it's a good idea to mix it with global flags. But no strong opinion here

Reply:
I'd like to keep it in the config, the alternative is to add another arg to everything that uses it

Reply:
Actually, log level is only required in root cmd, so there is no need to hold it in config.
And since we do not support `dryRun` for all operation, it's make sense to include only for cmds that support it. With the current implementation you have to handle the opposite case: return an error from cmds, which don't support it. Which, I think, easier to forget + can be misleading for the user (why do we suggest user the flag for the cmd, that is not supported?)

Reply:
the goal for dry run is that we can always see what will happen for any command, some are just not implemented yet

Reply:
I still think that it will be easy to miss during the implementation (which will lead to the case I described) and it's better to explicitly specify where `dry-run` is supported. At least until the days when we will support for most of the commands

Reply:
I've changed this to be command local